src/main/java/uk/gov/hmcts/ccd/domain/model/search/SearchResultViewColumn.java

```diff
@@ -48,4 +63,42 @@ public Integer getOrder() {
     public boolean isMetadata() {
         return metadata;
     }
+
+    public String getDisplayContextParameter() {
+        return displayContextParameter;
+    }
+
+    public Optional<DisplayContextParameter> getDisplayContextParameterOfType(DisplayContextParameterType displayContextParameterType) {
+        return DisplayContextParameter.getDisplayContextParameterOfType(getDisplayContextParameter(), displayContextParameterType);
+    }
+
+    public Optional<CommonField> getNestedField(String path) {
```

Comment:
looks like this method has significant logic. Can you add few unit tests to domain class please.